wrims_v2/wrims_v2/src/test/test_heclib7/Test_heclib7.java

```
@@ -95,9 +117,25 @@ public void writeData(String dvPath){
 			}
 		}catch (Exception e) {
 			System.out.println("Could not open dv file. "+e);
```

let the exception bubble out